src/app/downloads/table/summary-table.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { DownloadsTableModel, ChartType, CHARTTYPE_STACKED, CHARTTYPE_PODCAST } from '@app/ngrx';
+import * as dateFormat from '@app/shared/util/date/date.format';
+
+@Component({
+  selector: 'metrics-downloads-summary-table',
+  template: `
+    <div class="header row">
+      <div>Episode</div>
+      <div>Release Date</div>
+      <div class="charted">Downloads</div>
+      <div>All-Time</div>
+    </div>
+
+    <div class="row" *ngIf="podcastTableData">
+      <div>
+        <prx-checkbox *ngIf="showPodcastToggle; else podcastTitle"
+          small [checked]="podcastTableData.charted" [color]="podcastTableData.color"
+          (change)="toggleChartPodcast.emit($event)">
+          {{podcastTableData.title}}
+        </prx-checkbox>
+        <ng-template #podcastTitle><span class="title">{{podcastTableData.title}}</span></ng-template>
+        <span class="mobile title">{{podcastTableData.title}}</span>
```

Comment:
Should this be conditionally displayed as well?

Reply:
bleh, I don't love how this has 3 title fields 2 conditionally controlled by Angular, but the `.mobile.title` being displayed by CSS (along with the `.title:not(.mobile), prx-checkbox` being hidden.) It's confusing, and I didn't figure out a way around it. I added a comment about it and used a better class name. Here is that CSS: https://github.com/PRX/metrics.prx.org/pull/381/files#diff-3bc32d7b8aa81549c4ca5c265d15adf9R60

Reply:
Ah, the `:not` selector. That's what I was missing. Makes sense, if a bit awkward.

Reply:
definitely awkward, thought maybe the responsive table could be separate in the template... Changing flex-direction seemed to be fine except for hiding this functionality did come out a little messier.